cmd/admin/admin.go

```diff
@@ -94,3 +95,23 @@ func newAdmin(configFile string, dryRun bool) (*admin, error) {
 		log:    logger,
 	}, nil
 }
+
+// findActiveInputMethodFlag returns the single input method flag that was set
+// to a non-default value in setInputs map. If no flags or multiple flags are
+// set, it returns an error.
```

Comment:
This method doesn't know about default values or non-default values, it just knows about true and false.

```suggestion
// findActiveInputMethodFlag returns a single key from setInputs with a value of `true`,
// if exactly one exists. Otherwise it returns an error.
```